src/manager/event_handlers.ts

```diff
@@ -418,14 +420,19 @@ function refreshRoundedCorners(actor: ExtensionsWindowActor): void {
 
     const windowContentOffset = computeWindowContentsOffset(win);
 
+    const focused = win.appears_focused;
+    const border_color = focused
```

Comment:
```suggestion
    const borderColor = focused
```

The JS convention is to use lowerCamelCase for variable names, GNOME code uses snake_case for some reason so @yilozt probably went with snake case as well to be consistent with it, but I think it's better to just use the JS convention when possible